src/ontodia/data/sparql/graphBuilder.ts

```diff
@@ -17,16 +17,15 @@ const GREED_STEP = 150;
 
 export class GraphBuilder {
     constructor(
-        public dataProvider: DataProvider,
-        public endpointUrl: string
+        public dataProvider: SparqlDataProvider
```

Comment:
Look at ["allow trailing commas in parameter definitions and function calls"](http://www.2ality.com/2015/11/trailing-comma-parameters.html). You should see a TSLint warning here.